tests/layers/utils_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import numpy as np
+import tensorflow as tf
+from deepctr.layers.utils import Hash
+from tests.utils import layer_test
+try:
+    from tensorflow.python.keras.utils import CustomObjectScope
+except:
+    from tensorflow.keras.utils import CustomObjectScope
+
+
+@pytest.mark.parametrize(
+    'num_buckets,mask_zero,vocabulary_path,input_data,expected_output',
+    [
+        (3+1, False, None, ['lakemerson'], None),
+        (3+1, True, None, ['lakemerson'], None),
+        (3+1, False, "./tests/layers/vocabulary_example.csv", [['lake'], ['johnson'], ['lakemerson']], [[1], [3], [0]])
+    ]
+)
+def test_Hash(num_buckets, mask_zero, vocabulary_path, input_data, expected_output):
+    if not hasattr(tf, 'version') or tf.version.VERSION < '2.0.0':
```

Comment:
Hash Layer not work in tensorflow version under 2.0.0?

Reply:
`Hash` Layer can work in tensorflow version 1.x, it need run the code `sess.run(tf.table_initializer())`  while  initializing the model. I found the `layer_test` function which will create a model with the layer  on line https://github.com/shenweichen/DeepCTR/blob/0df401c5bbe2e00105aff62013ec687bea61774c/tests/utils.py#L207-L209 and predict the result without initializing the `table_initializer`.